config.js

```diff
@@ -373,7 +329,7 @@ if (argv.botid || argv.bot || argv.id || argv.minrankedhandicap || argv.maxranke
         return false;
     }
 
-    if (argv.ban) {
+    if (argv.ban && !argv.banranked && !argv.banunranked) {
```

Comment:
Make this `if (argv.ban}` so it'll add any global bans to the global ban list. People should be able to use --ban and --banranked and --banunranked. Just because you ban a few people totally doesn't mean you can't also ban someone else only from ranked games.

Reply:
ok i understand, these situations are indeed different because ban is a string !